### PR TITLE
Phoenix in an iframe

### PR DESCRIPTION
**Pull request by urfolomeus**
Bloggo about putting Phoenix in an iframe

### CODE REVIEW

**Review by paulanthonywilson — APPROVED** [review]
\o/

File: _posts/2019-01-21-putting-phoenix-in-an-iframe.md [hunk 1]
```diff
@@ -1,6 +1,6 @@
 ---
 layout: post
-title: How to deal with cross origin iframes in Phoenix
+title: Putting Phoenix in an iframe
```
Comment (paulanthonywilson) [inline _posts/2019-01-21-putting-phoenix-in-an-iframe.md]: Oh, like the new pithy title

File: _posts/2019-01-22-putting-phoenix-in-an-iframe.md [hunk 2]
```diff
@@ -10,6 +10,50 @@ We recently had cause to run a Phoenix application within an iframe on another a
...
+
+```elixir
+# controllers/allow_cross_origin_iframe.ex
+defmodule CultivateWeb.Example do
+  import Plug.Conn
+  def init(opts), do: opts
+
+  def call(conn, _options) do
+    conn
+    |> authenticate_source
```
Comment (paulanthonywilson) [inline _posts/2019-01-22-putting-phoenix-in-an-iframe.md]: Nice to have source.

Parenthesis needed here, though. `authenticate_source()`

Comment (paulanthonywilson) [reply _posts/2019-01-22-putting-phoenix-in-an-iframe.md]: (Actually never mind - I have it open)

**Review by paulanthonywilson — APPROVED** [review]

File: _posts/2019-01-22-putting-phoenix-in-an-iframe.md (hunk 2 above)
Comment (urfolomeus) [reply _posts/2019-01-22-putting-phoenix-in-an-iframe.md]: Thanks!

**Pull request merged**